djautotask/api_rest.py

```diff
@@ -335,7 +348,18 @@ def request(self, method, endpoint_url, body=None):
 
 
 class ContactsAPIClient(AutotaskAPIClient):
-    API = 'contacts'
+    API = 'Contacts'
 
     def get_contacts(self, next_url, *args, **kwargs):
         return self.fetch_resource(next_url, *args, **kwargs)
+
+
+class TicketsAPIClient(AutotaskAPIClient):
```

Comment:
I think nearly all other API client classes we have in django-connectwise and django-autotask use singular instead of plural for the class name. So `TicketAPIClient` instead of `TicketsAPIClient`

Reply:
As I remember, the naming here comes from API documents from AT. It was changed from singular to plural (SOAP to REST), then it was also applied to our REST API. It is now different with ones in CW, but it is the same with API docs in AT. What do you think about that?

![image](https://user-images.githubusercontent.com/73486314/115791436-3a57bc80-a37d-11eb-82c1-852cdfd791dc.png)

Reply:
ok. If that's what the docs are calling it now then I guess it's fine :man_shrugging:  . Thanks for pointing it out, I didn't realize that.